### ISSUE TITLE
WETH V1 Wormhole stuck

### ISSUE THREAD

**Issue by Rafanator** [solana-labs/oyster#353]
Hi, I have WETH stuck on Solana that I can't bridge back to Ethereum or sell in Solana. 

I had no issues bridging from ETH --> SOL. After bridging, I realised there is no WETH liquidity anyware. 

When I tried to bridge back to ETH, the tx would fail. Here are the tx hashes for the times this happened: 
- https://explorer.solana.com/tx/2tmGQLudXu6H4guQsM5PrcGhJoaLqyQhoaHZgx2x8rh5bfPw1N3cfHcjPMMUkpM3Ss8JaL9JXKgRTz5Um5Apk8DC
- https://explorer.solana.com/tx/4cfJ6Awr6Fob9YpMkDXaXxCgKBQZ1TzgiVkLdExmmMRsNqAqEsUiZzMmUXgwtEv1hBNdxEEQwmskEg5YE1nWjhwg
- https://explorer.solana.com/tx/cYuvFHZyRVReWBD7ksRavDR8Dt7PvjiJXxW2D4svswP5G5vcwz9mVXAaau2RgCvTfN8uYdf8wmW328dWJCtbFuo
- https://explorer.solana.com/tx/5ScDGTmsh3DV6xrDWgXEE4f1czoPbGYQF2epTN3qyvenKJNMvK6XXs4R2sTR1PgvaegLWd4ZWsUP6rR8vP1wZeLY

There are a few more tx's that failed with the same error, not necessary to include them. 

Now that v2 was released, I am told that I need to use v1 for this specific asset (pictured below):
![Screenshot 2021-09-20 at 11 35 58](https://user-images.githubusercontent.com/26544214/134011166-66fa1255-bf12-47e3-886f-c0f9760f813a.png)

 When I use v1, it does not reflect my current WETH balance on the v1 webapp:
![Screenshot 2021-09-20 at 11 35 21](https://user-images.githubusercontent.com/26544214/134011210-eb160fcc-5b8e-4d68-b165-d5659322c381.png)

When I fill in 1.5 WETH, I just get the icon that the tx is processing (pictured below), but I my wallet never pops up and requests a signature, so the tx never really happens on the chain.
![Screenshot 2021-09-20 at 11 44 07](https://user-images.githubusercontent.com/26544214/134011652-eb86dc69-cb5e-43df-884e-fd49bb211b46.png)

What can I do to solve this? Thanks! 

Wallet: Phantom
Address: 2A4nEr9WY7UeBeadsbueLcvz5rxTwMTi6UaJw6VjchT2
WETH balance: 1.5 
SOL Balance: 0.0279

**Comment by yamijuan** [comment]
Looks like you don't have enough SOL to pay the fee. Also I think you need to select the token called wWETH (wormhole) instead of that one.

**Comment by Rafanator** [comment]
How can I search assets? I don't see any wWETH, and I recall having it bridged it as just WETH: 
<img width="545" alt="Screenshot 2021-09-20 at 22 24 47" src="https://user-images.githubusercontent.com/26544214/134070544-6f19defc-4cad-48da-90df-77de7d748002.png">

**Comment by yamijuan** [comment]
looking at the transactions seems like the token is correct but you don't have enough SOL to complete the tx, iirc, you need 0.05 SOL at least

**Comment by Rafanator** [comment]
Yes, now i have more sOL, but the problem is still that the tx doesn't stage, and I dont see the 1.5 WETH on the website, as I included in my issue, I attach here as well:
![Screenshot 2021-09-20 at 11 35 21](https://user-images.githubusercontent.com/26544214/134072323-97587bf9-fcd2-4879-8a36-0eaac8078041.png)

^ No balance there, but the balance does show on my wallet and on v2. 

How do I fix that? Thanks for the help.

**Comment by yamijuan** [comment]
even tho the balance does not show up, can you try again? if doesn't work can you post a screenshot of the dev browser console?

**Comment by Rafanator** [comment]
After I had put 1.5
<img width="1437" alt="Screenshot 2021-09-20 at 22 59 38" src="https://user-images.githubusercontent.com/26544214/134074974-cc85c57b-9dbb-4389-a845-622bbe0f38b2.png">

Console close up:
<img width="557" alt="Screenshot 2021-09-20 at 22 59 13" src="https://user-images.githubusercontent.com/26544214/134075030-7dd19da7-bad4-4281-be8c-61e6fa1232fd.png">

**Comment by Rafanator** [comment]
Moar...
<img width="555" alt="Screenshot 2021-09-20 at 23 06 03" src="https://user-images.githubusercontent.com/26544214/134075788-bb98738e-f78d-4aa8-98cf-7735ced7d0b6.png">

**Comment by leoluk** [comment]
According to the failed transactions, you need more SOL on your fee payer wallet.

**Comment by Rafanator** [comment]
My problem is that the UI doesnt show the balance of wETH and now I cant even stage the transaciton. I included that in the pictures above.

**Comment by leoluk** [comment]
Okay, I'll leave this for @yamijuan

**Comment by Rafanator** [comment]
Please take a look at the comment above in this issue, where I detail that I don't see my balance reflected on the wormhole v1 website, but I do see it on my wallet. Screenshots are a few comments above ^^^

**Comment by Rafanator** [comment]
@yamijuan Hola Juan Diego. Me puedes ayudar porfavor? Can you please help me?

**Comment by evan-gray** [comment]
@Rafanator in your earlier messages it looks like your browser may have been rate limited by the Solana RPC node. A fix to cut down on the requests the initial page makes was released a couple days ago. If you try now do you still encounter the same issue? Have you tried transferring the funds again and ensured you have sufficient SOL for fees?

**Comment by Rafanator** [comment]
Hi, I tried just now, and a bunch of times earlier, I have the same.

No balance showing in the UI:
![Screenshot 2021-09-24 at 12 37 01](https://user-images.githubusercontent.com/26544214/134661409-be41074b-2048-47a6-bc5d-e789e175c66b.png)


If I type in the number, the image below pops up, but the tx is never staged and the wallet asks for no confirmation:
![Screenshot 2021-09-24 at 12 37 09](https://user-images.githubusercontent.com/26544214/134661454-9e8f98d9-0c51-430d-ada3-5d49cdb2cd27.png)

So basically, the same. Can i not interact directly with the smart contract? Can you not add wWETH to v2? 

Thanks for the help.

**Comment by evan-gray** [comment]
I appreciate your patience. You mention that your wallet doesn’t pop up, are there any error messages in the console when you try? I am less familiar with v1 but you are welcome to look at the source code for the v1 site in this repo. As for support in the v2 UI, none of the v1 tokens should be sent over the v2 bridge (it would result in double wrapped tokens). Some of the v1 tokens have migration pools available (accessible in the v2 ui) but it looks like v1 wWETH (wormhole) is not one of them. 🙁

**Comment by yamijuan** [comment]
once you connect your wallets, can you take a screenshot of the token list available? like all of them that show up in the dropdown.

**Comment by Rafanator** [comment]
Hi @evan-gray , here are two pictures showing the console logs:
![Screenshot 2021-09-24 at 18 25 16](https://user-images.githubusercontent.com/26544214/134709163-f010841d-fc3f-4302-96e6-353c62eacfbf.png)

![Screenshot 2021-09-24 at 18 25 25](https://user-images.githubusercontent.com/26544214/134709181-8520f933-3378-4b77-a152-d8783c12f306.png)

Hi @yamijuan here is the screenshot of the token list available (only shows SOL and WETH):

![Screenshot 2021-09-24 at 18 25 39](https://user-images.githubusercontent.com/26544214/134709272-575be287-0978-4cf7-a517-26ef4718d10c.png)

**Comment by evan-gray** [comment]
@Rafanator have you tried adding more SOL to your account as @leoluk suggested? A quick look at your recent transactions from your wallet seem to have failed due to insufficient funds (to pay fees). 
`Transfer: insufficient lamports 7435240, need 9131520`
e.g. https://explorer.solana.com/tx/2tmGQLudXu6H4guQsM5PrcGhJoaLqyQhoaHZgx2x8rh5bfPw1N3cfHcjPMMUkpM3Ss8JaL9JXKgRTz5Um5Apk8DC

**Comment by Rafanator** [comment]
@evan-gray Please read the full thread. Those old tx hashes where when I had not enough SOL, and the tx would still stage and let me sign.

Now, the tx doesn't stage and doesn't let me sign. I will try with 1 full SOL now, but I am almost sure it is not the problem...

**Comment by yamijuan** [comment]
I don't really understand why is that happening to you... Quick question, before, when you were able to sign the transaction, did the balance show up correctly? or has it never?

**Comment by Rafanator** [comment]
Yes. The balance used to be there before v2 was realeased, and it was back then when all the example tx's that failed would fail.

**Comment by Rafanator** [comment]
Ok, I added more balance, and I still have the same problem. 

Is this the correct url? http://v1.wormholebridge.com/#/move?from=SOL&token=WETH

Please help, this has taken ages, no chance for a refund if this doesn't work? You can see my wWETH on chain...
